spec/serve.js

```diff
@@ -65,6 +52,45 @@ describe('production server', function () {
 
   it('should create build files', function () {
     var fileNames = fs.readdirSync(buildDir);
-    assert(fileNames.length = 2);
+    assert.equal(fileNames.length, 3);
```

Comment:
In order to understand those numbers, you have to have the mock application's structure and content in mind at all times. 

I think this could be improved by either adding an explanatory comment or changing the assertions to check the actual files.